modules/ROOT/pages/prerequisites/index.adoc

```diff
@@ -19,19 +19,23 @@
 
 {description}
 
+Note that oCIS is based on a highly dynamic interoperable _microservices_ concept, even when it is delivered as single binary. Of course this software has to run on hardware, but its ability to adapt to various and maybe dynamic load scenarios across physical boundaries means that you can setup your physical environment flexibly using modern technology.
+
+The following sections are fundamental to understanding the general prerequisites, but their implementation is very dependent on your use-case. To support your decision about the environment, we recommend reading the documents about  xref:architecture/index.adoc[Architecture and Concepts] and xref:availability/index.adoc[Availability, Redundancy and Load Distribution] first.
+
 == Hardware
 
-The minimum hardware requirements depend on the usage scenario you have in mind for your oCIS instance. You need at least hardware that runs a Linux based operating system preferrable 64bit.
+The minimum hardware requirements depend on the usage scenario you have in mind for your oCIS instance. You need at least hardware that runs a Linux based operating system, preferably 64bit.
 
 * For simple testing purposes, you can even use a laptop or Raspberry Pi 4 (ARM64) with a minimum of 512MB of RAM.
 * For more intense testing purposes, the recommended start hardware is a multi-core processor with at least 4GB of RAM.
-* For productive environments, CPU and RAM are the main limiting factors. A clear guideline for CPU's cant be given at the moment as there are too many influencing factors. See xref:ram-considerations[RAM Considerations] for memory related factors.
+* For production environments, CPU and RAM are the main limiting factors. A clear guideline for CPU's cant be given at the moment as there are too many influencing factors. See xref:ram-considerations[RAM Considerations] for memory and xref:bandwidth-considerations[Bandwidth Considerations] for network related factors.
```

Comment:
```suggestion
* For production environments, CPU and RAM are the main limiting factors. A clear guideline for CPUs can't be provided at this time since there are too many influencing factors. See xref:ram-considerations[RAM Considerations] for memory and xref:bandwidth-considerations[Bandwidth Considerations] for network-related factors.
```